features/websocket.yml

```
@@ -1,7 +1,8 @@
-draft_date: 2024-07-11
-name: WebSockets Standard
-description: TODO
+name: WebSockets
+description: WebSockets provides a way to open a two-way communication channel between the user's browser and a server.
```

https://github.com/web-platform-dx/web-features/tree/main/docs#provides recommends against "provides" in most cases, and this is a case where I think it's simpler without it:

```suggestion
description: The `WebSocket` API opens a two-way communication channel between the user's browser and a server.
```

@ddbeck do you approve?